### ISSUE TITLE
Missing changelog entries for 104

### ISSUE THREAD

**Issue by rathann** [WebAssembly/binaryen#4444]
The changelog section for [104](https://github.com/WebAssembly/binaryen/blob/main/CHANGELOG.md?plain=1#L18) contains just one entry while the [list of commits](https://github.com/WebAssembly/binaryen/compare/version_103...version_104) is much longer. Would it be possible to list the high-level changes and mention the [fix](https://github.com/WebAssembly/binaryen/commit/b1f6298ed8756bdc3336429c04b92ba58d000b49) for CVE-2021-45290 and CVE-2021-45293? Please have the Github security advisories updated as well. They're showing both affected and patched versions as "Unknown".

**Comment by kripken** [comment]
I'm curious to hear more about the importance of those CVEs, in your opinion. My perspective is that they are just minor miscellaneous bugs, because Binaryen is not used in a place where a DoS attack is of concern. (It's also not used in a security-sensitive place where an exploit would be worrying, like a browser or an OS kernel.) But maybe I'm wrong?

Regardless, a PR with additional details would definitely be welcome.

**Comment by rathann** [comment]
You are not wrong, I guess. I asked only for completeness sake, because I got bug reports opened downstream, against binaryen package in Fedora and EPEL and I haven't found an upstream changelog when releasing an update.

**Comment by kripken** [comment]
I see. Well, perhaps we should get more into the habit updating the changelog as we land things. I think that's probably a good idea in general.

**Comment by tlively** [comment]
In general, we have been pretty good about updating the changelog for new releases. PRs are always welcome adding anything we may have missed as well. I'll close this issue, but if anyone wants to further discuss changelogs, please open a new issue.
